src/main/java/uk/gov/pay/connector/gateway/worldpay/wallets/WorldpayWalletAuthorisationHandler.java

```diff
@@ -41,13 +41,21 @@ public WorldpayWalletAuthorisationHandler(GatewayClient authoriseClient, Map<Str
 
     private GatewayOrder buildWalletAuthoriseOrder(WalletAuthorisationGatewayRequest request) {
 
-        return aWorldpayAuthoriseWalletOrderRequestBuilder(request.getWalletAuthorisationData().getWalletType())
+        boolean is3dsRequired = request.getGatewayAccount().isRequires3ds();
```

Comment:
Can we use `is3dsRequired()` instead of `isRequires3ds()`?
```suggestion
        boolean is3dsRequired = request.getGatewayAccount().is3dsRequired();
```

Reply:
Yes - could I do that in a follow up PR because renaming the method in the GatewayAccountEntity will cause a lot of extra modified files to be added into this PR

Reply:
Yes